static/app/views/replays/detail/network/details/components.tsx

```diff
@@ -0,0 +1,89 @@
+import {Fragment, ReactNode, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {KeyValueTable, KeyValueTableRow} from 'sentry/components/keyValueTable';
+import {IconChevron} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+export const Indent = styled('div')`
+  padding-left: ${space(4)};
+`;
+
+const NotFoundText = styled('span')`
+  color: ${p => p.theme.subText};
+  font-size: ${p => p.theme.fontSizeSmall};
+`;
+
+export function keyValueTablOrNotFound(
+  data: undefined | Record<string, string>,
+  notFoundText: string
+) {
+  return data ? (
+    <StyledKeyValueTable noMargin>
+      {Object.entries(data).map(([key, value]) => (
+        <KeyValueTableRow key={key} keyName={key} value={<span>{value}</span>} />
+      ))}
+    </StyledKeyValueTable>
+  ) : (
+    <Indent>
+      <NotFoundText>{notFoundText}</NotFoundText>
+    </Indent>
+  );
+}
+
+const SectionTitle = styled('dt')``;
+
+const SectionData = styled('dd')`
+  font-size: ${p => p.theme.fontSizeExtraSmall};
+`;
+
+const ToggleButton = styled('button')`
+  background: ${p => p.theme.background};
+  border: 0;
+  color: ${p => p.theme.headingColor};
+  font-size: ${p => p.theme.fontSizeSmall};
+  font-weight: 600;
+  line-height: ${p => p.theme.text.lineHeightBody};
```

Comment:
just side thoughts, wonder if we need a "naked" button that's styled with reset type css

Reply:
we probably do, it's not the first time on this page we've brought it a bunch of resets. semantics!